platform/admin/ui/src/main/webapp/less/application/app-card.less

```diff
@@ -25,6 +25,8 @@
   .panel-body {
     padding: 0px @app-card-overlay-width 0px 0px; // reset bootstrap
     height: 100%;
+    display: flex;
```

Comment:
Is flex supported on all of they browsers we support. For some reason I thought it wasn't. @andrewkfiedler would know.

Reply:
@clockard Is there somewhere I can find a list of supported browsers?

Reply:
I don't know that there is an exhaustive list but the docs say the minimum browser requirements are IE 10. I think for this particular change, IE would be the only browser of concern.